src/client/executor.rs

```diff
@@ -16,44 +16,28 @@ use super::{options::ServerAddress, session::TransactionState, Client, ClientSes
 use crate::{
     bson::Document,
     change_stream::{
```

Comment:
I can already tell rustfmt is going to complain on this. My local rustfmt is on a different version and autoformats on save (sigh)